paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -456,23 +457,26 @@ class ConvMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
           platform::MKLDNNMemDesc(dst_tz, dst_dt, chosen_memory_format);
 
       // create a conv primitive descriptor and save it for usage in backward
+      // TODO(lidanqing): We use relu post-op instead of brelu post-op cause
+      // mkldnn v0.18 does not support INT8 brelu post-op. Use code in /**/ when
+      // v0.20 is enabled
       if (bias) {
         bias_tz = paddle::framework::vectorize2int(bias->dims());
         auto bias_md = platform::MKLDNNMemDesc(bias_tz, memory::data_type::s32,
                                                memory::format::x);
 
         conv_pd = ConvFwdPrimitiveDesc(
             src_md, weights_md, bias_md, dst_md, strides, paddings,
-            mkldnn_engine, fuse_relu, fuse_residual_conn, false /*fuse_brelu*/,
-            0.0 /*fuse_brelu_threshold*/, output_shift_scale, sum_scale,
-            is_test);
+            mkldnn_engine, fuse_relu || fuse_brelu /*fuse_relu*/,
+            fuse_residual_conn, false /*fuse_brelu*/, fuse_brelu_threshold,
+            output_shift_scale, sum_scale, is_test);
 
       } else {
-        conv_pd = ConvFwdPrimitiveDesc(src_md, weights_md, dst_md, strides,
-                                       paddings, mkldnn_engine, fuse_relu,
-                                       fuse_residual_conn, false /*fuse_brelu*/,
-                                       0.0 /*fuse_brelu_threshold*/,
-                                       output_shift_scale, sum_scale, is_test);
+        conv_pd = ConvFwdPrimitiveDesc(
+            src_md, weights_md, dst_md, strides, paddings, mkldnn_engine,
+            fuse_relu || fuse_brelu /*fuse_relu*/, fuse_residual_conn,
+            false /*fuse_brelu*/, fuse_brelu_threshold, output_shift_scale,
+            sum_scale, is_test);
```

Comment:
Could you combine `ConvFwdPrimitiveDesc` two functions, `if (bias) = false`, pass `bias_md=nullptr` into it.

Reply:
> Could you combine `ConvFwdPrimitiveDesc` two functions, `if (bias) = false`, pass `bias_md=nullptr` into it.

Ok, I will do like this.

Reply:
I merge your pr at first, and you can refine in next PR!

Reply:
> I merge your pr at first, and you can refine in next PR!

Thank you! I start to refine now and will check the refined code through tests. I just update that it could be mkldnn v1.0 that will be used by baidu. Both support this op and both will be released after paddle release 1.5.

Reply:
Got it. But the refined code PR could be created before paddle release 1.5?

Reply:
> Got it. But the refined code PR could be created before paddle release 1.5?

Yes, will be before 1.5